package.json

```diff
@@ -46,7 +47,9 @@
     ]
   },
   "dependencies": {
-    "classnames": "^2.2.5",
+    "@emotion/core": "^11.0.0",
```

Comment:
I think this should be a `peerDependencies` alongside the other @emotion packages.
I guess this package should have limited control over which emotion version it uses therefore it should be a peer.

Also if you add it to `peerDependencies` you must also add it to `devDependencies`

Reply:
Also if someone wants to use the base components emotion must be optional (although as a peerDependency it will throw a warning)

Reply:
Moved emotion packages to peer dependencies, removed @emotion/core, seeing as @emotion/react is equivalent